master/getting-started/kubernetes/upgrade/setup.md

```diff
@@ -17,40 +18,39 @@ target etcdv3 server. The host must be running one of the following:
 
 ### About installing calico-upgrade
 
-We distribute `calico-upgrade` in both binary and container formats. Refer to the
-instructions that correspond to your preferred format.
+You can install `calico-upgrade` as either of the following. 
 
 - [Binary](#installing-calico-upgrade-as-a-binary)
 
 - [Container](#installing-calico-upgrade-as-a-container)
 
 ### Installing calico-upgrade as a binary
 
-1. Log into the host, open a terminal prompt, and navigate to the location where
-you want to install the binary. 
+1. [Download the `calico-upgrade` binary appropriate to your operating system](https://github.com/projectcalico/calico-upgrade/releases/latest).
 
-   > **Tip**: Consider navigating to a location that's in your `PATH`. For example, 
-   > `/usr/local/bin/`.
+   > **Tip**: Consider downloading it to a location that's already in your `PATH`. For example, 
+   > `/usr/local/bin/`. Alternatively, add its location to your `PATH`. Once it's in your `PATH`, 
+   > you can invoke it without having to prepend its location.
    {: .alert .alert-success}
 
-1. [Download the `calico-upgrade` binary appropriate to your operating system](https://github.com/projectcalico/calico-upgrade/releases/tag/v0.1.0).
-
 1. Set the file to be executable.
 
    ```
    chmod +x calico-upgrade
    ```
-
-   > **Note**: If the location of `calicoctl` is not already in your `PATH`, move the file
-   > to one that is or add its location to your `PATH`. This will allow you to invoke it
-   > without having to prepend its location.
-   {: .alert .alert-info}
    
-1. Congratulations! You've installed `calico-upgrade`. Continue to [Configuring calico-upgrade](#configuring-calico-upgrade)
+1. Congratulations! You've installed `calico-upgrade`. Continue to [Configuring calico-upgrade](#configuring-calico-upgrade).
 
 ### Installing calico-upgrade as a container
```

Comment:
I wonder if we're just providing too many options. Maybe we should just stick to the binary approach and be done with it. No need to give users a gun to shoot themselves in the foot with.

Reply:
Makes the rest of the process flow better too.

Reply:
We are distributing it as a container for some reason...I am guessing some people want to run it inside of one? I am not sure. This is following on from the `calicoctl` which is a similar situation. It makes the docs more complex (especially for calicoctl which has a reduction in functionality that's easy to miss) but it seems that there is an audience for the container method? Personally, I would like to see both just as binaries but I do not know about the users.

Reply:
Yea we ship both but we don't need to document the container approach here.

We could really just leave it as a note "We also distribute the calico-upgrade tool as the `calico/upgrade` docker container."

That would be enough.

Reply:
This does offer the command which they can copy and paste and ensures that they pull down the correct version of the docker container.